src/commands/create-run.yml

```diff
@@ -16,16 +16,23 @@ parameters:
     default: "./project-config.json"
     description: |
       Specify path to the project config JSON file. Default is "./project-config.json".
-  dryRun:
-    description: when true it runs configuration validations without creating run
+  test-args:
+    type: string
+    default: ""
+    description: |
+      Pass custom arguments to the test script in order to trigger a particular set of tests
```

Comment:
Could it be more explicit about the expected arguments? Like file name or test name to make sure the user will know what it is?